File: Service/ClientCare/RMA.Service.ClientCare.Contracts/Entities/Broker/BrokerageRepresentativeRequest.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

using System.Collections.Generic;

namespace RMA.Service.ClientCare.Contracts.Entities.Broker
{
    public class BrokerageRepresentativeRequest
    {
        public string FspNumber { get; set; }
        public List<string> RepresentativeIdNumbers { get; set; }
        public BrokerageTypeEnum BrokerageType { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/clientcare/src/app/broker-manager/models/brokerage-representative-request.ts
import { BrokerageTypeEnum } from './../../../../../shared-models-lib/src/lib/enums/brokerage-type-enum';
export class BrokerageRepresentativeRequest {
  fspNumber: string;
  representativeIdNumbers: string[];
  BrokerageType :BrokerageTypeEnum
}
